api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -545,8 +545,8 @@ public ResponseEntity<Void> confirmPublications() {
   }
 
   @Override
-  public ResponseEntity<Void> extendInitialCreditExpiration() {
-    initialCreditsService.extendInitialCreditsExpiration(userProvider.get());
-    return ResponseEntity.noContent().build();
+  public ResponseEntity<Profile> extendInitialCreditExpiration() {
```

Comment:
Changed extendInitialCreditExpiration to return profile, so that the profile object can be used to update state in the UI without having to make a separate call to the profile endpoint.